[2025-01-06T17:09:16.560+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:09:16.601+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.clearing_table_task manual__2025-01-06T17:09:10.884749+00:00 [queued]>
[2025-01-06T17:09:16.624+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.clearing_table_task manual__2025-01-06T17:09:10.884749+00:00 [queued]>
[2025-01-06T17:09:16.626+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:09:16.660+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): clearing_table_task> on 2025-01-06 17:09:10.884749+00:00
[2025-01-06T17:09:16.670+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=259) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:09:16.674+0000] {standard_task_runner.py:72} INFO - Started process 260 to run task
[2025-01-06T17:09:16.674+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'clearing_table_task', 'manual__2025-01-06T17:09:10.884749+00:00', '--job-id', '825', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpjn9k6_di']
[2025-01-06T17:09:16.677+0000] {standard_task_runner.py:105} INFO - Job 825: Subtask clearing_table_task
[2025-01-06T17:09:16.765+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.clearing_table_task manual__2025-01-06T17:09:10.884749+00:00 [running]> on host 97bbb29a7501
[2025-01-06T17:09:16.922+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='clearing_table_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:09:10.884749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:09:10.884749+00:00'
[2025-01-06T17:09:16.924+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:09:16.947+0000] {sql.py:278} INFO - Executing: TRUNCATE STG.adv;
[2025-01-06T17:09:16.962+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T17:09:16.975+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T17:09:16.982+0000] {sql.py:553} INFO - Running statement: TRUNCATE STG.adv;, parameters: None
[2025-01-06T17:09:17.013+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:09:17.015+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=clearing_table_task, run_id=manual__2025-01-06T17:09:10.884749+00:00, execution_date=20250106T170910, start_date=20250106T170916, end_date=20250106T170917
[2025-01-06T17:09:17.052+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:09:17.069+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
